deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Net45.wxs

```diff
@@ -6,31 +6,37 @@
   <Fragment>
     <ComponentGroup Id="Files.Managed.Net45" Directory="net45">
       <Component Win64="$(var.Win64)">
+        
```

Comment:
Adding the condition in the template causes those blank lines. Couldn't find an easy way to remove them.